app/models/job_application.rb

```diff
@@ -0,0 +1,13 @@
+class JobApplication < ApplicationRecord
+  belongs_to :company
+  belongs_to :user
+
+  validates :position_title, presence: true
+  validates :date_applied, presence: true
+  validates :status, presence: true
+  validates :notes, presence: true
```

Comment:
Im not sure if notes should be required for a job application? Not all users will want to post a note in every situation

Reply:
Good catch, in hindsight I agree, not everyone wants to submit notes. I will correct this and resubmit.